.drone.yml

```diff
@@ -33,12 +36,48 @@ steps:
   commands:
   - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
   - docker login -u="ukhomeofficedigital+drone" -p=$${DOCKER_PASSWORD} quay.io
-  - docker tag file-vault-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/file-vault:$${DRONE_COMMIT_SHA}
-  - docker push quay.io/ukhomeofficedigital/file-vault:$${DRONE_COMMIT_SHA}
+  - docker tag $${IMAGE_REPO}:$${DRONE_COMMIT_SHA} $${IMAGE_URL}/$${IMAGE_REPO}:$${DRONE_COMMIT_SHA}
+  - docker push $${IMAGE_URL}/$${IMAGE_REPO}:$${DRONE_COMMIT_SHA}
   when:
     branch: master
     event: [ push, pull_request ]
 
+ # Trivy Security Scannner
+- name: scan_packages
+  pull: always
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
+  resources:
+    limits:
+      cpu: 1000
+      memory: 1024Mi
+  environment:
+    IMAGE_NAME: file-vault:${DRONE_COMMIT_SHA}
+    SEVERITY: MEDIUM,HIGH,CRITICAL
+    FAIL_ON_DETECTION: false
+    IGNORE_UNFIXED: true
+# ALLOW_CVE_LIST_FILE: hof-services-config/file-vault/trivy-cve-exceptions.txt
+  when:
+    branch: master
+    event: [ push, pull_request, tag ]
```

Comment:
should we include merge as well?

Reply:
@DivineTaminangHO - Please could you comment on this

Reply:
Push to a branch in Drone is same as Merge. We don't have an event merge.
We can remove tag from events which we are not using. Please use COA as reference @DivineTaminangHO